spec/cocina/models/validators/associated_name_validator_spec.rb

```diff
@@ -149,25 +149,15 @@
     end
   end
 
-  context 'when valid DRO with name title group' do
-    let(:clazz) { Cocina::Models::DRO }
-
-    let(:props) { { description: desc_props } }
-
-    it 'does not raise' do
-      validate
-    end
-  end
-
   context 'when invalid DRO with name title group' do
     let(:clazz) { Cocina::Models::DRO }
 
     let(:contributor_name) { 'not Smith, John' }
 
     let(:props) { { description: desc_props } }
 
-    it 'raises' do
-      expect { validate }.to raise_error(Cocina::Models::ValidationError)
+    it 'does not validate' do
+      expect { validate }.not_to raise_error(Cocina::Models::ValidationError)
```

Comment:
If the test for 'does not validate' why does it not expect a validation error here?

Reply:
I agree this is unclear. Refactoring tests.